src/main/scala/dev/atedeg/UbiquitousScaladocKeys.scala

```diff
@@ -0,0 +1,15 @@
+package dev.atedeg
+
+import sbt.*
+
+trait UbiquitousScaladocKeys {
+
+  val ubiquitousScaladoc: TaskKey[Unit] =
+    taskKey[Unit]("Task for creating a Ubiquitous Language markdown table out of a Scaladoc html file")
+
+  val usSourceHtmlDir: SettingKey[File] =
```

Comment:
Why the SettingKey's name start with `us`?
`sourceHtmlDir` is already taken inside SBT?